Makefile

```diff
@@ -45,7 +45,7 @@ libEGM$(LIB_EXT): libEGM
 libEGM: .FORCE libProtocols$(LIB_EXT) libENIGMAShared$(LIB_EXT)
 	$(MAKE) -C CommandLine/libEGM/
 
-EMAKE_TARGETS = .FORCE ENIGMA libProtocols$(LIB_EXT) libEGM$(LIB_EXT)
```

Comment:
No need for this target here as it is already inside ``$(LIB_PFX)compileEGMf$(LIB_EXT)``.